cnf-certification-test/manageability/suite.go

```diff
@@ -52,15 +53,18 @@ var _ = ginkgo.Describe(common.ManageabilityTestKey, func() {
 })
 
 func testContainersImageTag(env *provider.TestEnvironment) {
-	badContainers := []string{}
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
 	for _, cut := range env.Containers {
 		logrus.Debugln("check container ", cut.String(), " image should be tagged ")
-		if cut.ContainerImageIdentifier.Tag == "" {
-			badContainers = append(badContainers, cut.String())
+		if cut.IsTagEmpty() {
+			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, fmt.Sprintf("Container %s is missing image tag(s)", cut.String()), false))
```

Comment:
no need to add the container id -> fmt.Sprintf("Container %s is missing image tag(s)", cut.String()) -> "Container is missing image tag(s)"

Reply:
Fixed.